### ISSUE TITLE
Black eschar

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9960]
**Preferred term label:**
Black eschar

**Synonyms**
Tache noire; black spot

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Boutonneuse fever

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000793
- label: Eschar
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:16455730
- def: A type of wound characterized by dry, black, hard necrotic tissue.
- comment: Since necrotic tissue can also harbour pathogenic organisms, removal of such tissue helps to prevent wound infection. Necrotic tissue and slough should be debrided with a scalpel so that the wound bed can be accurately assessed and facilitate healing. Eschar may be adherent to the wound bed, making debridement with a scalpel difficult. Further debridement, as part of wound management, may be required using other techniques.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9960
- synonyms: Tache noire(NONE)|Black eschar(NONE)|Black spot(NONE)
